.gitignore

```diff
@@ -1,13 +1,21 @@
-node_modules
-dist
-
-.nyc_output
-coverage
-
+*.log
+.DS_Store
+.coverage
+.coverage-integration
+.coverage-unit
+.editorconfig
```

Comment:
why `.editorconfig` is included?

Reply:
To be consistent across all of the repos by keeping it out. See Lautaro's comment [here](https://github.com/poetapp/random/pull/134#issuecomment-414787838).

Reply:
thanks!!!